apps/passport-client/components/screens/AddSubscriptionScreen.tsx

```diff
@@ -86,15 +107,15 @@ export function AddSubscriptionScreen(): JSX.Element {
   useEffect(() => {
     if (!self || userForcedToLogout) {
       clearAllPendingRequests();
-      const stringifiedRequest = JSON.stringify(url ?? "");
+      const stringifiedRequest = JSON.stringify(url);
       setPendingAddSubscriptionRequest(stringifiedRequest);
       const emailParameter = suggestedEmail
         ? `&email=${encodeURIComponent(suggestedEmail)}`
         : "";
       if (!self) {
```

Comment:
should we redirect in all cases here, not just when `!self`?

Reply:
Hmm, it does seem to make sense to redirect on both conditions, not just one.